samples/drivers/touch/README.rst

```diff
@@ -0,0 +1,31 @@
+.. zephyr:code-sample:: touch
+   :name: touch
+
+   Basic touchscreen sample.
```

Comment:
This needs a proper name and description should be worded as if completing the sentence "This code sample shows how to ..."

```suggestion
.. zephyr:code-sample:: draw_touch_events
   :name: Draw touch events
   :relevant-api: input_events display_interface

   Visualize touch events on a display.
```

Reply:
Done.